pkg/keymanagementprovider/azurekeyvault/provider.go

```diff
@@ -244,7 +282,7 @@ func initializeKvClient(ctx context.Context, keyVaultEndpoint, tenantID, clientI
 
 // Parse the secret bundle and return an array of certificates
 // In a certificate chain scenario, all certificates from root to leaf will be returned
-func getCertsFromSecretBundle(ctx context.Context, secretBundle kv.SecretBundle, certName string) ([]*x509.Certificate, []map[string]string, error) {
+func getCertsFromSecretBundle(ctx context.Context, secretBundle kv.SecretBundle, certName, enabled string) ([]*x509.Certificate, []map[string]string, error) {
```

Comment:
nit: I would recommend to define a bool var for enabled.

Reply:
That can certainly be done, but will require some refactoring from a  [string]string map to a [string]interface. :)